src/scene/scene_phongmaterial.js

```diff
@@ -211,6 +211,9 @@ pc.extend(pc, function () {
             this.emissiveMapTint = false;
             this.emissiveIntensity = 1;
 
+            this.refraction = 0;
+            this.refractionIor = 1.0 / 1.5; // approx. (air ior / glass ior)

```

Comment:
This property name is completely opaque to me. Suggested alternative: refractionIndex